README.md

```diff
@@ -165,17 +173,24 @@ Make sure to replace the parameters accordingly.
 10. Trigger the DAG with the following:
 
 ```bash
-airflow trigger_dag -c  "{\"params\": { \"resource_id\": \"my-res-id-123\", \
-\"schema_fields_array\": [ \"field1\", \"field2\" ], \
-\"csv_input\": \"/path/to/my.csv\", \
-\"json_output\": \"/path/to/my.json\" } }" \
-ckan_api_load_single_step
+airflow trigger_dag ckan_api_load_single_step \
+ --conf='{ "resource_id": "my-res-id-123", "schema_fields_array": [ "field1", "field2" ], "csv_input": "/path/to.csv", "json_output": "/path/to.json" }'
 ```
 
 Do not forget to properly replace the parameters with your data and properly escape the special characters.
 Alternatively, you can just run the DAG with the `airflow run` command.
 
-### Example 3: Local file to CKAN DataStore using Postgres
+
+`api_ckan_load_single_node` also works for remote CKAN instances. Just set up your Airflow `CKAN_SITE_URL` variable accordingly.
+
+
+#### Multiple-node DAG
+
+`ckan_api_load_multiple_steps` does the same steps of `api_ckan_load_single_node`, but it uses multiple nodes (tasks). You can repeat the steps of the previous section and run `ckan_api_load_multiple_steps`.
+
+
+
```

Comment:
On the other hand while triggering the DAG without sending parameters in CLI:
`airflow trigger_dag ckan_api_load_multiple_steps`
and setting DAG parameters directly in `args` in code seems working:
```
args = {
    'owner': 'airflow',
    'start_date': datetime.now() - timedelta(hours=4),
    'params': { 
        "resource_id": "be9f63d3-40c1-4261-acb1-232bf675da64",
        "schema_fields_array": "[ 'id', 'full_text', 'FID' ]",
        "csv_input": "/home/michael/airflow/example.csv",
        "json_output": "/home/michael/airflow/my.json"
    }
}
```

Reply:
The idea is to be able to pass the params on the trigger. Like, imagine someone just grabbing our dags - we may want something transparent where users do not need to modify the code

Reply:
Sure, that was just to get things working, actually the double quotes make the difference :100: